### PR TITLE
LVNOD-157-Refactor-all-reducers

### PR DESCRIPTION
**Pull request by denisbodnar**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-157)

## Code reviewers

- [ ] @yarvanya
- [ ] @AlbaMontis
- [ ] @shyyka

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## CHECK LIST
- [ ]  СI passed
- [ ]  PR meets all conventions

### CODE REVIEW

File: app/src/components/EventsModalForm/editEventActions.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { API, request } from 'src/helper';
+
+const EDIT_EVENT_SUCCESS = 'EDIT_EVENT_SUCCESS';
+const EDIT_EVENT_FAILURE = 'EDIT_EVENT_FAILURE';
+
+export const editEventSuccess = res => {
+  return {
```
Comment (Vovk-Nazar) [inline app/src/components/EventsModalForm/editEventActions.js]: => ({
your object
})

check all PR

File: app/src/containers/Event/eventActions.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const FETCH_EVENT_BY_ID_FULFILLED = 'FETCH_EVENT_BY_ID_FULFILLED';
+const FETCH_EVENT_BY_ID_REJECTED = 'FETCH_EVENT_BY_ID_REJECTED';
+
+export const fetchEventById = eventId => dispatch => {
+  return request()
+    .get(`${API.URL}/api/event/${eventId}`)
+    .end((err, res) => {
+      if (err) {
+        dispatch({
```
Comment (Vovk-Nazar) [inline app/src/containers/Event/eventActions.js]: create const with action and call this action here

check all action files

File: app/src/containers/Event/inviteActions.js [hunk 3]
```diff
@@ -55,9 +36,9 @@ export function getEmails(eventId) {
           });
       });
   };
-}
+};
 
-export function deleteGuest(eventId, userId) {
+export const deleteGuest = (eventId, userId) => {
   return dispatch => {
```
Comment (Vovk-Nazar) [inline app/src/containers/Event/inviteActions.js]: fix this and check all action

File: app/src/containers/Event/inviteActions.js [hunk 4]
```diff
@@ -69,9 +50,9 @@ export function deleteGuest(eventId, userId) {
...
       });
   };
-}
+};
 
-export function saveEmails(emails, eventId) {
+export const saveEmails = (emails, eventId) => {
   const data = {emails, eventId};
   return dispatch => {
     request()
```
Comment (Vovk-Nazar) [inline app/src/containers/Event/inviteActions.js]: fix 59 line

**Review by Vovk-Nazar — CHANGES_REQUESTED** [review]

**Review by shyyka — DISMISSED** [review]

File: app/src/containers/Events/eventsListReducer.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+export const eventListReducer = {
+  RETRIEVE_EVENTS: (state, action) => ({
+    ...state
```
Comment (4asof4ik) [inline app/src/containers/Events/eventsListReducer.js]: Unnecessary  action

File: app/src/containers/Contacts/contactActions.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  show: show
+});
+
+export const messageModal = message => ({
+  type: MESSAGE_MODAL,
+  message: message
+});
+
+export const contactRequest = () => ({type: CONTACT_REQUEST});
+
```
Comment (4asof4ik) [inline app/src/containers/Contacts/contactActions.js]: Unnecessary  action

File: app/src/containers/Contacts/contactActions.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+export const contactInfo = data => dispatch => {
+  dispatch(contactRequest());
```
Comment (4asof4ik) [inline app/src/containers/Contacts/contactActions.js]: And here

File: app/src/components/Login/loginActions.js [hunk 8]
```diff
@@ -47,45 +17,39 @@ export function loginUserSuccess(token, userId, profileId) {
...
+};
 
-export function loginUser(email, password) {
+export const loginUser = (email, password) => {
   const user = {email, password};
   let token;
   let userId;
   let profileId;
   return dispatch => {
     dispatch(loginUserRequest());
```
Comment (4asof4ik) [inline app/src/components/Login/loginActions.js]: Unnecessary

File: app/src/components/Login/loginActions.js [hunk 9]
```diff
@@ -47,45 +17,39 @@ export function loginUserSuccess(token, userId, profileId) {
...
+  message
+});
 
-export function loginUserRequest() {
-  return {
-    type: LOGIN_USER_REQUEST
-  };
-}
+export const loginUserRequest = () => ({
+  type: LOGIN_USER_REQUEST
```
Comment (4asof4ik) [inline app/src/components/Login/loginActions.js]: Drop this action

File: app/src/containers/Profile/profileActions.js [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+  dispatch(retrieveProfileRequest());
+  return request()
+    .get(`${API.URL}/api/user/${userId}`)
+    .end((error, res) => {
+      !error && dispatch(retrieveProfileSuccess(res));
+    });
+};
+
+export const updateProfile = profile => dispatch => {
+  dispatch(updateProfileRequest());
```
Comment (4asof4ik) [inline app/src/containers/Profile/profileActions.js]: Drop

File: app/src/containers/Profile/profileActions.js [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+  type: UPDATE_PROFILE
+});
+
+const updateProfileSuccess = res => ({
+  type: UPDATE_PROFILE_SUCCESS,
+  payload: res.body
+});
+
+export const retrieveProfile = userId => dispatch => {
+  dispatch(retrieveProfileRequest());
```
Comment (4asof4ik) [inline app/src/containers/Profile/profileActions.js]: Drop And action too

File: app/src/containers/Profile/profileActions.js [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+const retrieveProfileRequest = () => ({
+  type: RETRIEVE_PROFILE
+});
+
+const retrieveProfileSuccess = res => ({
+  type: RETRIEVE_PROFILE_SUCCESS,
+  payload: res.body
+});
+
+const updateProfileRequest = () => ({
```
Comment (4asof4ik) [inline app/src/containers/Profile/profileActions.js]: Drop

File: app/src/containers/Profile/profileActions.js [hunk 13]
```diff
@@ -0,0 +1,45 @@
+import { API, request } from 'src/helper';
+
+const RETRIEVE_PROFILE = 'RETRIEVE_PROFILE';
+const RETRIEVE_PROFILE_SUCCESS = 'RETRIEVE_PROFILE_SUCCESS';
+const UPDATE_PROFILE = 'UPDATE_PROFILE';
+const UPDATE_PROFILE_SUCCESS = 'UPDATE_PROFILE_SUCCESS';
+
+const retrieveProfileRequest = () => ({
```
Comment (4asof4ik) [inline app/src/containers/Profile/profileActions.js]: Drop

File: app/src/containers/Events/eventsListActions.js [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+  type: RETRIEVE_EVENTS_SUCCESS,
+  payload: res.body,
+});
+
+export const retrieveEventsRequest = () => ({
+  type: RETRIEVE_EVENTS
+});
+
+export const retrieveEvents = () => dispatch => {
+  dispatch(retrieveEventsRequest());
```
Comment (4asof4ik) [inline app/src/containers/Events/eventsListActions.js]: Drop And action

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by yarvanya — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

File: app/src/redux/configureStore.js [hunk 15]
```diff
@@ -3,17 +3,17 @@ import { routerMiddleware } from 'react-router-redux';
...
 const middleware = [routerMiddleware(browserHistory), thunk];
 
-if (process.env.NODE_ENV !== 'production') {
-  middleware.push(createLogger());
-}
+(process.env.NODE_ENV !== 'production') && middleware.push(createLogger());
 
-export default function configureStore(initialState) {
+const configureStore = initialState => {
   return createStore(
```
Comment (4asof4ik) [inline app/src/redux/configureStore.js]: drop return

**Review by 4asof4ik — APPROVED** [review]

**Review by yarvanya — APPROVED** [review]

**Pull request merged**